.github/workflows/main.yml

```diff
@@ -66,7 +66,7 @@ jobs:
   tests:
     name: Tests
     runs-on: ubuntu-latest
-    container: docker.io/library/golang:${{ matrix.go }}-alpine
+    container: quay.io/app-sre/golang:${{ matrix.go }}-alpine
```

Comment:
@jan-zmeskal 

just a sanity check, each image in this matrix exists in the app-sre repo correct?

Reply:
Yes, last Friday (December 4) I checked that all the tags in the matrix are present in https://quay.io/repository/app-sre/golang. As far as I can tell, all the tags are mirrored expect those containing "windows" and "nanoserver" in their name.